packages/legend-application-data-cube/src/stores/LegendDataCubeDuckDBEngine.ts

```diff
@@ -45,9 +46,14 @@ export class LegendDataCubeDuckDBEngine {
   // https://duckdb.org/docs/guides/meta/describe.html
   private static readonly COLUMN_NAME = 'column_name';
   private static readonly COLUMN_TYPE = 'column_type';
+  private static readonly TABLE_NAME = 'table_name';
   // Options for creating csv using papa parser: https://www.papaparse.com/docs#config
   private static readonly ESCAPE_CHAR = `'`;
   private static readonly QUOTE_CHAR = `'`;
+  private static dbReferenceMap: Map<
```

Comment:
Don't think we need to make this static. 

Also let's rename.


```suggestion
  private static catalog: Map<
```